docs/markdown/Using Pants/troubleshooting.md

```diff
@@ -281,4 +281,4 @@ Setting up pants to run with Python executables provided by [setup-python](https
 /home/ubuntu/.cache/python-tools/Python/3.11.3/x64/bin/python3.11: error while loading shared libraries: libpython3.11.so.1.0: cannot open shared object file: No such file or directory
 ```
 
-One of the workaround to fix this issue is setting up python tool cache files at `/opt/hostedtoolcache` directory. This is the default path which `setup-python` action uses to download relevant files on hosted GitHub actions runner. Overriding tool cache download directory can be achieved by following [setup-python documentation](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#linux).
\ No newline at end of file
+One of the workaround to fix this issue is setting up python tool cache files at `/opt/hostedtoolcache` directory. This is the default path which `setup-python` action uses to download relevant files on hosted GitHub actions runner. Overriding tool cache download directory can be achieved by following [setup-python documentation](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#linux).
```

Comment:
What changed here?

Reply:
I had to check locally to figure that one out, tho.. :p 

![image](https://github.com/pantsbuild/pants/assets/72965/e202c1cc-8406-4a11-9819-bfc5bfaa5001)